### PR TITLE
Add WNP 85 for /de/ (Fixes #9856)

### LINKED ISSUE #9856: WNP 85 for DE
## Description

(Mozilla access only)
Brief: https://docs.google.com/document/d/17HXO98YR4oeD9LJLtQ2mkmbJqp-O_6ZSfYAclsPfJwM/edit?usp=sharing
Copy doc: https://docs.google.com/document/d/1iGqg7yi9W5eXMyR9AKAi-gA3fD2HeP5RKYor6P3s1p4/edit#

---

#### Success Criteria
- [ ] WNP 85 is live in production for DE before Jan 26th.

### PR DESCRIPTION
**Pull request by alexgibson**
## Description

http://localhost:8000/de/firefox/85.0/whatsnew/all/

## Issue / Bugzilla link
#9856

## Testing
Successful test run: https://gitlab.com/mozmeao/www-config/-/pipelines/245020276

### CODE REVIEW

**Comment by alexgibson** [comment]
This is ready for code review, and needs to be live on Monday 25th ready for the 85 launch the next day.

Marking as do-not-merge for now whilst legal takes a look at the copy.

File: bedrock/firefox/templates/firefox/whatsnew/whatsnew-mobile-de.html [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+  {{ css_bundle('firefox_whatsnew_mobile_de') }}
+{% endblock %}
+
+{% block content %}
+<main role="main" class="content-wrapper mzp-t-firefox">
+  {% include 'firefox/whatsnew/includes/header.html' %}
+
+  <section class="wnp-content-main">
+    <div class="mzp-l-content">
+    <img class="wnp-main-image" src="{{ static('protocol/img/logos/firefox/browser/logo-md.png') }}" alt="">
```
Comment (maxxcrawford) [inline bedrock/firefox/templates/firefox/whatsnew/whatsnew-mobile-de.html:33]: Reviewing the rendered HTML, I'm not getting the [high-res version](https://github.com/mozilla/protocol-assets/blob/main/logos/firefox/logo-md-high-res.png) of the hero logo. Is it missing a `data-high-res="true"` attribute? 

An alternative would be to to use [the SVG version](https://github.com/mozilla/protocol-assets/blob/main/logos/firefox/logo.svg), and make sure the `.wnp-main-image` class has a max-width.

**Review by maxxcrawford — CHANGES_REQUESTED** [review]
Requesting one change (low-res image issue) but looks good otherwise! 

I am calling out an additional issue, but it may be out of scope for this PR: The error text is in english when trying to submit the Send to Device form. Is this something that can be fixed here? If not, I can file a seperate bug. 

image

**Comment by alexgibson** [comment]
> I am calling out an additional issue, but it may be out of scope for this PR: The error text is in english when trying to submit the Send to Device form. Is this something that can be fixed here? If not, I can file a seperate bug.

The tooltip is a browser feature, not part of the web page itself, so the language depends on your browser settings.

**Comment by alexgibson** [comment]
@maxxcrawford thanks for the review, updated with the high res image.

**Review by maxxcrawford — APPROVED** [review]
LGTM! 🚀

**Comment by maxxcrawford** [comment]
@alexgibson Added the ⚠️ DNM label, just in case.

**Comment by alexgibson** [comment]
Got the green light to merge / push this.

**Pull request merged**